scripts/slurm-worker.py

```diff
@@ -0,0 +1,105 @@
+import math
+import subprocess
+import time
+
+from codalab.lib.formatting import parse_duration
```

Comment:
Document what this file does.  Perhaps rename this file to `nlp-slurm-worker.py`?  Or we can call `srun` directly instead of going through `nlprun` to make it more generic...